qa/run_sc_tests.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+set -e -o pipefail
+
+#Log stdout to file, use '| tee /dev/fd/3' to also log to console
+#Must match logfile location referenced in sc_test_framework
+exec 3>&1 1>>"${BASH_SOURCE%/*}/sc_test.log" 2>&1
+
+#Check Environment Variables have been set
+if [[ -z "$BITCOINCLI" ]]; then
+    echo "Environment Variable: BITCOINCLI not set" | tee /dev/fd/3
+    exit 1
+fi
+if [[ -z "$BITCOIND" ]]; then
+    echo "Environment Variable: BITCOIND not set" | tee /dev/fd/3
+    exit 1
+fi
+if [[ -z "$SIDECHAIN_SDK" ]]; then
+    echo "Environment Variable: SIDECHAIN_SDK not set" | tee /dev/fd/3
+    exit 1
+fi
+
+# parse args
+for i in "$@"; do
+  case "${i}" in
+    -extended)
+      EXTENDED="true"
+      shift
+      ;;
+    -exclude=*)
+      EXCLUDE="${i#*=}"
+      shift
+      ;;
+    -split=*)
+      SPLIT="${i#*=}"
+      shift
+      ;;
+    *)
+      # unknown option/passOn
+      passOn+="${i} "
+      ;;
+  esac
+done
+
+#Run the tests
+testScripts=(
+    'websocket_server_fee_payments.py'
```

Comment:
Pls, check the list of tests to be executed.
I saw the changes in the last PR, so no it is only one test in the list.